pilot/pkg/credentials/kube/multicluster.go

```diff
@@ -43,7 +43,7 @@ func NewMulticluster(configCluster cluster.ID) *Multicluster {
 	return m
 }
 
-func (m *Multicluster) ClusterAdded(cluster *multicluster.Cluster, _ <-chan struct{}) error {
+func (m *Multicluster) ClusterAdded(cluster *multicluster.Cluster, _ <-chan struct{}, update bool) error {
```

Comment:
what does update do

Reply:
This tells us to set the update flag on the controller so we know to do the audit on the sync.   But per Steve's comment above we should refactor so both handleUpdate is only called during the update and handleAdd only called during an add.   We would remove this flag as part of that refactor.